tox.ini

```
@@ -2,8 +2,8 @@
 skipsdist=True
 envlist=
     lint,
-    py{36}-dj{111,20,22}-wag{23},
-    py{36,38}-dj{22}-wag{28}
+    py{36}-dj{111,22}-wag23,
+    py{36,38}-dj22-wag29
```

These can still be [factors](https://tox.readthedocs.io/en/latest/config.html?highlight=factor#generating-environments-conditional-settings), even if there's just one of them.

```suggestion
    py{36}-dj{111,22}-wag{23},
    py{36,38}-dj{22}-wag{29}
```

---

Done